ere-guests/stateless-validator/reth/pico/Cargo.toml

```diff
@@ -19,7 +19,10 @@ reth-primitives-traits = { workspace = true, features = [
 reth-evm-ethereum = { workspace = true }
 reth-chainspec = { workspace = true }
 
-revm = { workspace = true, default-features = false, features = ["bn"] }
+revm = { workspace = true, default-features = false, features = [
+    "kzg-rs",
```

Comment:
The cause of block failure was a runtime failure of revm detecting not `c-kzg` nor `kzg-rs` were enabled. The reason why `kzg-rs` wasn't enabled was because of this Pico [compilation error](https://pastebin.com/raw/JjCLW51w) I found yesterday. This was confirmed by the Brevis team.

But, I tried an unodcumented precompile patch of `kzg-rs` directly that was merged [some hours ago](https://github.com/brevis-network/kzg-rs/commit/7f034a1a0f9328e7c2ae667a5b6cb98bf6ab2a79), and this fixes the compilation problem thus the runtime problem too.

Reply:
That branch doesn't compile